packaging/installer/netdata-updater.sh

```diff
@@ -584,6 +720,25 @@ while [ -n "${1}" ]; do
   elif [ "${1}" = "--tmpdir-path" ]; then
     NETDATA_TMPDIR_PATH="${2}"
     shift 2
+  elif [ "${1}" = "--enable-auto-updates" ]; then
+    AUTO_UPDATE_TYPE="$(echo "${2}" | tr '[:upper:]' '[:lower:]')"
+    case "${AUTO_UPDATE_TYPE}" in
+      systemd|interval|crontab)
+        enable_netdata_updater "${AUTO_UPDATE_TYPE}"
+        exit $?
+        ;;
+      "")
+        enable_netdata_updater ""
+        exit $?
+        ;;
+      *)
+        error "Unrecognized updater type ${2} requested. Supported types are 'systemd', 'interval', and 'crontab'."
+        exit 1
+        ;;
+    esac
```

Comment:
How about moving this code into enable_netdata_updater()?

```bash
elif [ "${1}" = "--enable-auto-updates" ]; then
    enable_netdata_updater "${2}"
    exit $?
elif [ "${1}" = "--disable-auto-updates" ]; then
    disable_netdata_updater
    exit $?

Reply:
Agreed, that would be much cleaner.